### PR TITLE
README Updates

### PR DESCRIPTION
**Pull request by MikeHathaway**
- Add `MULTICHAIN_STRATEGY.md` readme
- Split `README.md` into `TOKEN.md` and `GRANT_FUND.md` readmes.

### CODE REVIEW

File: MULTICHAIN_STRATEGY.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Multichain Strategy
+
+## Design
+
+Ajna deployments to each chain are sovereign and independant. Each chain will utilize the token address of Ajna on that chain for the burn-and-buy mechanism. The grant fund will remain entirely on L1, with only L1 token delegates able to vote.
+
+### **Rollups**
+
+Ajna pool factories will be deployed on every Ethereum L2 rollup, with each pool pointing at the token address created by each rollups canonical bridge. This token address is deterministic and tied to the L1 address, so it doesn't matter who bridges the token first. This behaviour is the same across rollups (Arbitrum, Optimism, Zksync).
```
Comment (EdNoepel) [inline MULTICHAIN_STRATEGY.md]: - How will we guarantee we'll deploy to _every_ L2 rollup?
- possessive form: _address created by each **rollup's** canonical bridge_

File: MULTICHAIN_STRATEGY.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+Due to the lower safety of sidechain bridges, Ajna tokens must first be wrapped in the `BurnWrapper.sol` contract. Pools in the sidechains will point to the sidechain token address created by mapping the BurnWrapped tokens to the sidechain token. Each sidechain will have it's own BurnWrapper instance. This design prevents double-spends due to sidechain bridges being compromised and releasing L1 tokens incorrectly, artifically increasing the token supply available for burn-and-buy. 
+
+#### Polygon 
+
+Tokens must be mapped to the polygon sidechain prior to bridging. This mapping will create a unique address for the token on the sidechain. Once this address has been created, it can be referenced permissionlessly in either of the Polygon POS, or Polygon Plasma bridges.
+
+#### Binance
+
+Requires a binance account to use the canonical binance bridge. Third-party bridges rely on the binance bridge as a source of truth. Don't necessarily have to be listed to use the bridge, but unclear.
```
Comment (EdNoepel) [inline MULTICHAIN_STRATEGY.md]: should capitalize _Binance_

File: src/GRANT_FUND.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# Ajna Grant Coordination Fund
+
+As a decentralized protocol with no external governance, Ajna requires a sustainable mechanism by which to coordinate ecosystem growth in a scalable and decentralized way. The grant coordination mechanism is premised upon the idea that the AJNA token has value from the buy and burn mechanism. Upon launch, the Ajna community will be given a fixed portion of the total AJNA token supply, which will be held and distributed by the Grant Coordinartion Fund.
```
Comment (EdNoepel) [inline src/GRANT_FUND.md]: spelling: _Grant **Coordination** Fund_

File: src/GRANT_FUND.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+- https://doseofdefi.substack.com/p/multichain-governance-how-can-daos
+- https://github.com/gnosis/zodiac
+- https://ethereum-magicians.org/t/eip-draft-multi-chain-governance/9284
+
+*General Governance Research*
+- https://github.com/D3LAB-DAO/Governor-C
+
+*Potential Vote Counting structures*
+- https://github.com/bokkypoobah/BokkyPooBahsRedBlackTreeLibrary
+
```
Comment (EdNoepel) [inline src/GRANT_FUND.md]: extra blank lines on 56 and 64

**Review by EdNoepel — COMMENTED** [review]
Proofread, left some comments.
Great doco.

File: MULTICHAIN_STRATEGY.md (hunk 1 above)
Comment (MikeHathaway) [reply MULTICHAIN_STRATEGY.md]: We don't really have any way to guarantee it. The current consensus AFAIK is to either add additional chains via the standard funding mechanism, or do it as part of the launch process

**Review by EdNoepel — APPROVED** [review]

**Pull request merged**